using `ember init` to update projects made with this blueprint fails if you're already removed ember-fetch, etc

error:
```
🚧  Installing packages... This might take a couple of minutes.
pnpm: Installed @embroider/core@unstable, @embroider/vite@unstable and 5 other packages

🚧  Installing packages... This might take a couple of minutes.
Command failed with exit code 1: pnpm remove ember-fetch broccoli-asset-rev ember-cli-app-version ember-cli-clean-css ember-cli-dependency-checker ember-cli-sri ember-cli-terser
 ERR_PNPM_CANNOT_REMOVE_MISSING_DEPS  Cannot remove 'ember-fetch', 'broccoli-asset-rev', 'ember-cli-app-version', 'ember-cli-clean-css', 'ember-cli-dependency-checker', 'ember-cli-sri', 'ember-cli-terser': no such dependencies found

```

why use `ember init`?

ember-cli-update requires a valid ember-cli-update.json config -- so how do you get your first ember-cli-update.json config? :see_no_evil:

--- a/index.js
+++ b/index.js
@@ -41,20 +41,28 @@ module.exports = {
     });
 
     let uninstallTask = this.taskFor('npm-uninstall');
-    await uninstallTask.run({
-      'save-dev': true,
-      verbose: false,
-      packages: [
-        'ember-fetch',
-        'broccoli-asset-rev',
-        'ember-cli-app-version',
-        'ember-cli-clean-css',
-        'ember-cli-dependency-checker',
-        'ember-cli-sri',
-        'ember-cli-terser',
-      ],
-      packageManager: options.packageManager,
-    });
+    const packages = [
+      'ember-fetch',
+      'broccoli-asset-rev',
+      'ember-cli-app-version',
+      'ember-cli-clean-css',
+      'ember-cli-dependency-checker',
+      'ember-cli-sri',
+      'ember-cli-terser',
+    ];
+
+    for (const package of packages) {
+      try {
+        await uninstallTask.run({
+          'save-dev': true,
+          verbose: false,
+          packages: [package],
+          packageManager: options.packageManager,
+        });
+      } catch (e) {
+        console.log(`Could not uninstall ${package}`);
+      }
+    }
   },
 
   async afterInstall(options) {

--- a/index.js
+++ b/index.js
@@ -59,7 +59,7 @@ module.exports = {
           packages: [package],
           packageManager: options.packageManager,
         });
-      } catch (e) {
+      } catch {
         console.log(`Could not uninstall ${package}`);
       }
     }

